src/deployment-guides/tko-aws.md

```diff
@@ -233,14 +233,27 @@ After doing the network configuration, complete the steps described in this sect
 	<!-- /* cSpell:disable */ -->
 	```bash
 	scp -i tkgkp.pem tanzu-cli-bundle-linux-amd64.tar  kubectl-linux-v1.21.2+vmware.1.gz ubuntu@$(jq -r '.Reservations[0].Instances[0].PublicIpAddress' $WORKING_DIR/instance_jb_started):/home/ubuntu
+	```
+	<!-- /* cSpell:enable */ -->
+
+7. Connect to the jumpbox and start port forwarding
+
+	Note that the command shown below assumes that no process is currently
```

Comment:
AS we do not know what may be running on 8080, should we recommend that they simply find an unused local port?

Reply:
I'd be in favor of saying something like "If port 8080 is already in use on your system, choose another available port"

Reply:
The reason I chose to say "stop what is currently running on 8080" is because the place where the port is used (in browsing to `http://localhost:8080`) is later on in the text. That means there would need to be a clarification there too. While it should be obvious to someone who's used to port forwarding that that is what you need to do, I've encountered enough people for whom it's completely new that I was trying to avoid the problem.